tests/combine_ecsv_files.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+
+"""Simple script to combine the ecsv files from individual test runs into a single monolithic file for easier
+review.
+
+"""
+import argparse
+from astropy.io import ascii
+from astropy.table import Table, vstack
+import glob
+import os
+import sys
+import traceback
+
```

Comment:
Based on PEP8, imports should be grouped (with blank lines between the groups) starting with the most general.  I would suggest the order to be (each on a separate line):  argparse, glob, os, sys, traceback, (blank line), astropy.io, astropy.table.

Reply:
Issue resolved by commit [d59bc2f](https://github.com/spacetelescope/drizzlepac/pull/351/commits/d59bc2f61546923b44c1bc7b06549e73e9a42747